examples/cli0.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+"""
+GRPC Client with command line options
+
+Usage:
+    client.py -i <router_IP> -p <port> -u <user> -pw <password> -r <rpc> [--file <json_file>] [--t <timeout>]
+    client.py -i <router_IP> -p <port> -u <user> -pw <password> -r <rpc> [<show_cmd>] [--t <timeout>]
+    client.py (-h | --help)
+
+Options:
+    -h --help    Show this help
+    --version	 Show version
+    -i           IP address of the router which the client will connect to
+    -p           Router gRPC server will be running on some TCP port, generally 57400
```

Comment:
Are we doing this file?

Reply:
cli0.py is a test file.  we can throw it away